src/ragas/testset/__init__.py

```diff
@@ -1,4 +1,11 @@
+from ragas.testset.persona import Persona, PersonaList
 from ragas.testset.synthesizers.generate import TestsetGenerator
 from ragas.testset.synthesizers.testset_schema import Testset, TestsetSample
 
-__all__ = ["TestsetGenerator", "Testset", "TestsetSample"]
+__all__ = [
+    "TestsetGenerator",
+    "Testset",
+    "TestsetSample",
+    "Persona",
+    "PersonaList",
```

Comment:
no need to add it here

Reply:
the reason being that users should manually import all of the dependencies from the corresponding modules - but it is also harder when customizing to be honest.